types/utils.go

```diff
@@ -7,27 +7,30 @@ import (
 	"time"
 
 	"github.com/cosmos/cosmos-sdk/types/kv"
+	"github.com/mailru/easyjson"
 	log "github.com/tendermint/tendermint/libs/log"
 )
 
-// SortedJSON takes any JSON and returns it sorted by keys. Also, all white-spaces
+// SortJSON takes any JSON and returns it sorted by keys. Also, all white-spaces
 // are removed.
 // This method can be used to canonicalize JSON to be returned by GetSignBytes,
 // e.g. for the ledger integration.
 // If the passed JSON isn't valid it will return an error.
 func SortJSON(toSortJSON []byte) ([]byte, error) {
 	var c interface{}
-	err := json.Unmarshal(toSortJSON, &c)
+	err := easyjson.Unmarshal(toSortJSON, c)
 	if err != nil {
 		return nil, err
 	}
-	js, err := json.Marshal(c)
+	rawBytes, err := easyjson.Marshal(c)
```

Comment:
```go
// SortJSON takes any JSON and returns it sorted by keys.
```

I couldn't find anything in [the docs for mailru/easyjson](https://pkg.go.dev/github.com/mailru/easyjson) that suggests the JSON it produces will be sorted, by keys or otherwise. Can you point me to where that's guaranteed?